libraries/botbuilder-ai/src/qnamaker-utils/generateAnswerUtils.ts

```diff
@@ -60,20 +60,21 @@ export class GenerateAnswerUtils {
     public async queryQnaServiceRaw(endpoint: QnAMakerEndpoint, question: string, options?: QnAMakerOptions): Promise<QnAMakerResults> {
         const url: string = `${ endpoint.host }/knowledgebases/${ endpoint.knowledgeBaseId }/generateanswer`;
         var queryOptions: QnAMakerOptions = { ...this._options, ...options } as QnAMakerOptions;
-        
+
         queryOptions.rankerType = !queryOptions.rankerType ? RankerTypes.default : queryOptions.rankerType;
         this.validateOptions(queryOptions);
 
         var payloadBody = JSON.stringify({
             question: question,
+            strictFiltersJoinOpertionType: queryOptions.strictFiltersJoinOperator,
```

Comment:
in Qna Service Json payload (left side), it should be still strictFiltersCompoundOperationType